### DISCUSSION TITLE
How to run workflows manual only?

### DISCUSSION THREAD

**Discussion by ZedZipDev** [community/community#58135] [Actions] [answered]
### Select Topic Area

Question

### Body

Is there any settings do not run automatically but manual only?

**Comment by airtower-luna** [comment, marked as answer]
See here: https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow

Mind that *any* event configured in the `on` section will trigger a workflow, so if you want a workflow to run *only* manually, it needs to have a `workflow_dispatch` trigger *only*.
